src/components/AnonymousChat/child/ChannelAnonymousSubImage.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
 
+import dimen from '../../../utils/dimen';
 import {colors} from '../../../utils/colors';
 
 const ChannelAnonymousSubImage = ({anonPostNotificationUserInfo = null}) => {
   const styles = StyleSheet.create({
     postNotificationImage: {
-      width: 24,
-      height: 24,
-      borderRadius: 16,
+      width: dimen.normalizeDimen(24),
+      height: dimen.normalizeDimen(24),
+      borderRadius: dimen.normalizeDimen(12),
       position: 'absolute',
-      top: 30,
+      top: dimen.normalizeDimen(30 + 12),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application on different screen sizes. The use of `dimen.normalizeDimen` function to normalize dimensions based on the screen size is a good practice in React Native development. However, it's important to ensure that the `dimen.normalizeDimen` function is implemented correctly and tested thoroughly across different device sizes.
```diff
-      width: 24,
-      height: 24,
-      borderRadius: 16,
-      top: 30,
+      width: dimen.normalizeDimen(24),
+      height: dimen.normalizeDimen(24),
+      borderRadius: dimen.normalizeDimen(12),
+      top: dimen.normalizeDimen(30 + 12),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->